docs/card-present-payments.md

```diff
@@ -0,0 +1,54 @@
+# Card Present Payments
+## Project scope
+* Support payments using the Stripe SDK for US-based merchants that are already WCPay customers
+* The mobile apps should support:
+	* Discovering card readers.
+	* Connecting a card reader to a mobile device.
+	* Collecting payment from a customer on a cash-on-delivery order.
+	* Print a receipt
+	* E-mail a receipt.
+	* Update a card reader firmware.
```

Comment:
Just getting picky because you mentioned grammar 😅 but the first half of the list uses a different tense than the second one. For consistency, I'd either change "Discover card readers" or "Updating a card reader firmware"